src/attributes.adoc

```diff
@@ -65,7 +65,7 @@ endif::[]
 :non-csrrw-and:  <<CSRRWI>>, <<CSRRS>>, <<CSRRSI>>, <<CSRRC>> and <<CSRRCI>>
 
 :TAG_RESET_CSR: The tag of the CSR must be reset to zero. The reset values of the metadata and address fields are UNSPECIFIED.
-:REQUIRE_CRE_CSR: Access to this CSR is illegal if CRE for the current mode is zero (see <<section_cheri_disable>>).
+:REQUIRE_CRE_CSR: Access to this CSR is illegal if the effective CRE for the current privilege is zero (see <<section_cheri_disable>>).
```

Comment:
Unrelated to this change but it's a bit odd we use a macro for csrs and include for instructions.

Reply:
I've typically used includes for multi-line and variables for single line, but there's no hard rule there